### ISSUE TITLE
Lambda create failed nodejs10.x deprecated

### ISSUE THREAD

**Issue by rust84** [aws-samples/amazon-cognito-example-for-external-idp#83]
Deploy currently fails. The Lambda nodejs10.x runtime  has been deprecated. 

```
ExternalIdPDemo: creating CloudFormation changeset...
5:26:25 PM | CREATE_FAILED        | AWS::Lambda::Function        | PreTokenGenerationAEF6E6B3
Resource handler returned message: "The runtime parameter of nodejs10.x is no longer supported for creating or updating AWS Lambda functions. We recommend you use the ne
w runtime (nodejs14.x) while creating or updating functions. (Service: Lambda, Status Code: 400, Request ID: 657f8229-2c4d-42a3-bbd8-5246eb03f0c1, Extended Request ID: n
ull)" (RequestToken: 4505604b-0788-fff3-32d0-adc1845acf5f, HandlerErrorCode: InvalidRequest)

        new Function (/home/russell/repos/amazon-cognito-example-for-external-idp/cdk/node_modules/@aws-cdk/aws-lambda/lib/function.ts:453:35)
        \_ new BackendStack (/home/russell/repos/amazon-cognito-example-for-external-idp/cdk/src/cdk.ts:102:32)
        \_ Object.<anonymous> (/home/russell/repos/amazon-cognito-example-for-external-idp/cdk/src/cdk.ts:457:22)
        \_ Module._compile (node:internal/modules/cjs/loader:1097:14)
        \_ Object.Module._extensions..js (node:internal/modules/cjs/loader:1149:10)
        \_ Module.load (node:internal/modules/cjs/loader:975:32)
        \_ Function.Module._load (node:internal/modules/cjs/loader:822:12)
        \_ Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
        \_ node:internal/main/run_main_module:17:47
```

**Comment by rust84** [comment]
Have just seen there is an open merge request to fix this https://github.com/aws-samples/amazon-cognito-example-for-external-idp/pull/79

**Comment by jhart0** [comment]
This has now been merged in #79
